OpenRA.Mods.Common/Scripting/Properties/ProductionProperties.cs

```diff
@@ -32,14 +32,15 @@ public ProductionProperties(ScriptContext context, Actor self)
 		}
 
 		[ScriptActorPropertyActivity]
-		[Desc("Build a unit, ignoring the production queue. The activity will wait if the exit is blocked.")]
-		public void Produce(string actorType, string factionVariant = null)
+		[Desc("Build a unit, ignoring the production queue. The activity will wait if the exit is blocked.",
+			"If exitType is nil or unavailable, then an exit will be selected based on Buildable info.")]
+		public void Produce(string actorType, string factionVariant = null, string exitType = null)
```

Comment:
Following on from my original request, can you please change these `exitType`s to `productionType` for consistency with the rest of the code (note the one in the Desc above too)?

Reply:
There are several other traits/methods using `exitType`, but I won't comment on all of them.